app/controllers/organizations/pets_controller.rb

```diff
@@ -29,11 +29,21 @@ def show
   end
 
   def create
```

Comment:
I have not used a transaction on a controller ation before, but it made sense to me to create the pet and default pet tasks in one hit, or none at all. Curious what your thoughts are on doing this.

Reply:
I think a transaction is a good idea to avoid those weird invalid states.